glotaran/builtin/models/kinetic_image/irf.py

```diff
@@ -19,7 +19,7 @@ class IrfMeasured:
         "center": List[Parameter],
         "width": List[Parameter],
         "scale": {"type": List[Parameter], "allow_none": True},
-        "normalize": {"type": bool, "default": False},
+        "normalize": {"type": bool, "default": True},
```

Comment:
Changing the default value for a model option is a breaking change, unless before this was not doing what it was supposed to, but even then it should be clearly communicated. 
This is something for the release notes @s-weigand .

Reply:
THis not a change in default behaviour since the parameter was never used in the code. But everyone wants normalized irfs except a few cases, this is why I made it default `true`